services/movie.py

```diff
@@ -15,18 +20,22 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
     movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+    genres_ids: Optional[list] = None,
+    actors_ids: Optional[list] = None,
```

Comment:
The type annotations for `genres_ids` and `actors_ids` should specify the type of elements in the list, e.g., `Optional[list[int]]`, to improve code clarity and type safety.